SoftLayer/transports.py

```diff
@@ -108,6 +112,17 @@ def __call__(self, request):
 
             largs.insert(0, {'headers': headers})
 
+            interpreter_version = sys.version_info[0]
+
+            # In Python 2 xmlrpc_client.dumps(...) does not
+            # automatically perform base64 encoding.
```

Comment:
to tackle too many local variable restriction of landscape .. you could may be direclty compare sys.version_info[0] <3  

not sure who/why the limit of 15 local variables  is/was enforced